src/crypto-providers/trezorTypes.ts

```diff
@@ -70,9 +84,15 @@ export type TrezorPoolParameters = {
   relays: TrezorPoolRelay[];
   metadata: TrezorPoolMetadata;
 }
+
 export type TrezorTxCertificate = {
   type: number
   path?: Array<number>
   pool?: string
   poolParameters?: TrezorPoolParameters
 }
+
+export const enum TrezorCryptoProviderFeature {
+  MULTI_ASSET,
+  ALLEGRA, // this includes optional TTL and validity interval start
```

Comment:
I would not be afraid to declare those two "features" separately for the sake of clarity

Reply:
Refactored